src/Core/src/Handlers/WebView/WebViewHandler.Android.cs

```diff
@@ -0,0 +1,70 @@
+﻿using Android.Webkit;
+using Android.Widget;
+using static Android.Views.ViewGroup;
+using AWebView = Android.Webkit.WebView;
+
+namespace Microsoft.Maui.Handlers
+{
+	public partial class WebViewHandler : ViewHandler<IWebView, AWebView>, IWebViewDelegate
```

Comment:
I wonder if IWebVIewDelegate should be in a custom MauiAWebView. 

What do you think @PureWeen  ?

Reply:
Ok i moved to a custom WebVIew